### PR TITLE
Update quotes.ts

### PR DESCRIPTION
**Pull request by Praveen007-K**
### Additional context

---

### What is the purpose of this pull request? 

- [x] Add quotes
- [ ] Bug fix
- [ ] New Feature
- [ ] Add more tests
- [ ] Other

### Before submitting the PR, please make sure you do the following

- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123` or `closes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: src/quotes.ts [hunk 1]
```diff
@@ -248,9 +248,14 @@ export const quotes: Quote[] = [
 	},
 	{
 		quote: `Put your hand on a hot stove for a minute, and it seems like an hour. Sit with a pretty girl for an hour and it seems like a minute. That is relativity.`,
-		author: `Albert Einstein`,
+		author: `Paulo Coelho`,
```
Comment (AnilSeervi) [inline src/quotes.ts:251]: By `author`, I mean the one who actually said that and not the one who quoted it @Praveen007-K

**Review by AnilSeervi — CHANGES_REQUESTED** [review]
Can you look into why the checks failed and do the needed change?
I think its the apostrophe. You need to use the "right singe quote" for the test to pass.
While you are at it, can you also move the new one to the end of the list. Thank you.

**Review by Praveen007-K — COMMENTED** [review]
Changes made

**Comment by AnilSeervi** [comment]
> Changes made

I don't think the changes made any difference @Praveen007-K, the checks still seems to be failing.
Can you follow the [Contributing guidelines](https://github.com/AnilSeervi/inspirational-quotes/blob/main/CONTRIBUTING.md) properly and make the required changes

**Comment by AnilSeervi** [comment]
Closing due no response.

**Pull request closed without merging**